### PR TITLE
Update links & images, Archive RocketChat

### PR DESCRIPTION
**Pull request by 1somniak**
- I put the wiki prog and algo course website in archive because they are not used anymore.
- RocketChat is in archive too because school no longer have access. I dont know if I should just remove it or put it in archive.
- I update some old links like moodle and rosetta stone and image links like zeus and rosetta stone.

### CODE REVIEW

File: _data/links.yml [hunk 1]
```diff
@@ -200,11 +200,11 @@
     - 'background-position-x: right'
 
   - name: Rosetta Stone
-    href: https://he2.tellmemorecampus.com/
+    href: https://login.rosettastone.com/
     class: tile-small-wide
-    background_image: https://resources.rosettastone.com/assets/ss/221270--8920100/App_Themes/Default/Global/0101UnidentifiedHomePage/Images/r/logo_RS.png
+    background_image: https://upload.wikimedia.org/wikipedia/en/thumb/6/67/Rosetta_Stone_logo.svg/1920px-Rosetta_Stone_logo.svg.png
```
Comment (matiboux) [inline _data/links.yml:205]: No need for an oversized logo here.
```suggestion
    background_image: https://upload.wikimedia.org/wikipedia/en/thumb/6/67/Rosetta_Stone_logo.svg/320px-Rosetta_Stone_logo.svg.png
```
This reduces file size from 151 KB to 20 KB.

File: _data/archive.yml [hunk 2]
```diff
@@ -31,6 +31,37 @@
...
+    - 'background-color: #f6f6f6'
+    - 'background-size: 90%'
+
+ - name: Algo Cours (FR)
+    href: https://algo.infoprepa.epita.fr/
+    class: tile-small-wide text-dark
+    background_image: https://algo.infoprepa.epita.fr/skins/common/images/Algo_prepas_epita_wiki_logo.png
+    style:
+    - 'background-color: #fff'
+    - 'background-size: 90%'
```
Comment (matiboux) [inline _data/archive.yml:48]: Broken indentation here
```suggestion
  - name: Algo Cours (FR)
    href: https://algo.infoprepa.epita.fr/
    class: tile-small-wide text-dark
    background_image: https://algo.infoprepa.epita.fr/skins/common/images/Algo_prepas_epita_wiki_logo.png
    style:
    - 'background-color: #fff'
    - 'background-size: 90%'
```

**Review by matiboux — CHANGES_REQUESTED** [review]
Thank you for the changes. The website really needed someone to look over and update what's changed in the past year. 🙏

I've suggested fixes to some of your changes, please review them.

Please also remove commented elements if they are redundant with some from other pages (like with the copies you made, as you meant to move the originals).

As for "Rocket.Chat", it is fine to send it in the archive. That's where I sent other deprecated and replaced resources, so it's welcome there.

However, I wonder about the Algo/Prog wikis: is their content really irrelevant now?
These resources were not used either in my time in Prepa (the Algo wiki even less so than Prog), but their content was nevertheless useful to some students.
Is their content fully redundant with some other resources? If not, please leave them out of the archive. Then, if you wish, we can think about moving them instead in the "Documentation & Resources" page.

**Comment by 1somniak** [comment]
ok I have removed all comments and did the changes requested.
for the algo/prog wikis, they are not used, no one of my teachers mentionned them during my prepa and these websites have not been updated since years.
I think it is better if they go in archive file

**Comment by matiboux** [comment]
Hello. Firstly, thank you for the changes.

Secondly, please revert changes to algo/prog wikis links. I insist these resources can be of use to students.
After looking at the content, and asking to schoolmates, I still believe their content is relevant.

> no one of my teachers mentionned them during my prepa

That's to be expected: curriculum changed and these wikis are no longer course material.
However, their content is still relevant and the core content have not changed.
For some, it gives another perspective on studied topics. For others, it gives a way to dive deeper in these topics.

> these websites have not been updated since years

The courses at EPITA have evolved but the studied topics are mostly the same. Most of the content of these websites are timeless and still available: courses content, some slides, practical sessions subjects, etc.

In my opinion, even if less than 10% of students use these resources, it is better to keep these resources handy and available while they are still up online, than not.

**Comment by 1somniak** [comment]
Hello,
I understand, I have done the changes requested :)

**Review by matiboux — CHANGES_REQUESTED** [review]
Thank you. "Wiki Prog" is still missing.
Once fixed, this'll be all good.

**Review by matiboux — APPROVED** [review]
Thank you for the contribution!

**Comment by 1somniak** [comment]
Sorry, I thought I added it ^^'
It's the last one I hope

**Comment by matiboux** [comment]
> Sorry, I thought I added it ^^' It's the last one I hope

All good! Sorry for the ping-pong game.

**Pull request merged**